src/main/java/task64/WordSearch.java

```diff
@@ -0,0 +1,29 @@
+package task64;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+
+public class WordSearch {
+    public static void main(String[] args) {
+        String filePath = "src/main/resources64/data.txt";
+        String searchWord = "Java";
+        int wordCount = 0;
+
+        System.out.println("Search word: " + searchWord);
+
+        try (BufferedReader reader = new BufferedReader(new FileReader(filePath))) {
+            String line;
+
+
+            while ((line = reader.readLine()) != null) {
+                if (line.contains(searchWord)) {
+                    wordCount++;
+                    System.out.println("Word found " + wordCount + " times.");
```

Comment:
Realizuj to wyświetlenie dopiero jak policzysz wszystkie wystąpienia danego słowa.
Obecnie wyświetlasz ten tekst za każdym razem, gdy je odnajdziesz czego wynikiem będzie np.

Word found 1 times.
Word found 2 times.
Word found 3 times.
..
..
..
Word found N times.